README.md

```diff
@@ -24,50 +24,66 @@ It offers:
 
 <ul>
 <li>
-    🔧 Plug ‘n’ play composable pipelines for creating datasets for
+    🔧 Plug ‘n’ play composable pipelines for creating datasets for:
     <ul>
         <li>AI image generation model fine-tuning (Stable Diffusion, ControlNet)</li>
         <li>Large language model fine-tuning (LLaMA, Falcon)</li>
     </ul>
 </li>
 <li>
-🧱 Library of off-the-shelf reusable components for
+🧱 Off-the-shelf library of easily adaptable reusable components for:
     <ul>
-    <li>Extracting data from public sources such as Common Crawl, LAION, ...</li>
-    <li>Filtering on 
+    <li>
+    Data Filtering
     <ul>
-        <li>Content, e.g. language, visual style, topic, format, aesthetics, etc.</li>
-        <li>Context, e.g. copyright license, origin</li>
-        <li>Metadata</li>
+        <li>Content: language, visual style, topic, format, aesthetics, NSFW, etc.</li>
+        <li>Context: e.g. copyright license, origin, etc.</li>
+        <li>Deduplication</li>
+    </ul>
+    </li>
+    <li>
+    Data Enrichment
+    <ul>
+        <li>Captions</li>
+        <li>Segmentations</li>
+        <li>Embeddings</li>
+    </ul>
+    </li>
+    <li>
+    Data Transformation
+    <ul>
+        <li>Images: cropping, resizing, etc. </li>
+        <li>Text: chunking, indexing, etc. </li>
+    </ul>
+    </li>
+    <li>
+    Data retrieval
+    <ul>
+        <li>Common Crawl</li>
+        <li>LAION</li>
```

Comment:
Can we summarize every category of filtering on a single line? Just so the view is a bit more condensed.